site2/website/release-notes.md

```diff
@@ -7,6 +7,111 @@
 
 * [IO] If Function Authorization is enabled, users have to be given the source/sink entitlement to run them. See https://github.com/apache/pulsar/pull/7466
 
+### 2.6.1 &mdash; 2020-08-07 <a id=“2.6.1”></a>
+
+#### Fixes and Enhancements
```

Comment:
Since we have no second heading 4, I suggest that we remove this heading, and change heading 5 to heading 4.
You can change the current heading 4 into a sentence like "The following lists fixes and enhancements in 2.6.1 release."

We use heading to make structure clear, if we list heading 3, heading 4 and heading 5 here without context, we can improve the structure. For heading usage, refer to https://developers.google.com/style/headings